v0.5/p1/spec/EBNF.md

```diff
@@ -0,0 +1,38 @@
+# EBNF 
+
+```chatinput
+<fastn> ::= <comment>* <module_comment>? <item>*
+<item> ::=  <comment> | <section>
+
+
+<module_comment> ::= (<single_module_comment> <new_line>)* <single_module_comment> (<new_line> | <eof>)
+<single_module_comment> ::= <doc_comment_marker> <opt_whitespace> <sentence>? <opt_whitespace> 
```

Comment:
```suggestion
<single_module_comment> ::= <doc_comment_marker> <space>* <sentence>? <opt_whitespace> 
```

Should we have `opt_whitespace`?

Reply:
Should we include the `<newline> | <eof>` here only? And rename this to `<module_comment_line>`?